From b11bd93be67c5ad92e279f071a812234486c128b Mon Sep 17 00:00:00 2001
From: Ming Liu <liu.ming50@gmail.com>
Date: Thu, 13 Jun 2019 16:52:45 +0200
Subject: [PATCH] CMakeLists.txt: fix libzip name

Change libzip -> zlib, this is needed for building with OE.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Ming Liu <liu.ming50@gmail.com>
---
 libuuu/CMakeLists.txt | 2 +-
 uuu/CMakeLists.txt    | 5 ++---
 2 files changed, 3 insertions(+), 4 deletions(-)

diff --git a/libuuu/CMakeLists.txt b/libuuu/CMakeLists.txt
index 740d7ed..30324d0 100644
--- a/libuuu/CMakeLists.txt
+++ b/libuuu/CMakeLists.txt
@@ -5,7 +5,7 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 find_package(PkgConfig REQUIRED)
 pkg_check_modules(LIBUSB REQUIRED libusb-1.0>=1.0.16)
-pkg_check_modules(LIBZIP REQUIRED libzip)
+pkg_check_modules(LIBZIP REQUIRED zlib)
 find_package(Threads)
 
 include_directories(${LIBUSB_INCLUDE_DIRS} include)
diff --git a/uuu/CMakeLists.txt b/uuu/CMakeLists.txt
index a2d8c87..1ca8013 100644
--- a/uuu/CMakeLists.txt
+++ b/uuu/CMakeLists.txt
@@ -5,8 +5,7 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 
 find_package(PkgConfig REQUIRED)
 pkg_check_modules(LIBUSB REQUIRED libusb-1.0>=1.0.16)
-pkg_check_modules(LIBZIP REQUIRED libzip)
-pkg_check_modules(LIBZ REQUIRED zlib)
+pkg_check_modules(LIBZIP REQUIRED zlib)
 find_package(Threads)
 
 
@@ -27,7 +26,7 @@ set(LSTS
 	sd_burn_all.lst
 )
 
-link_directories(${CMAKE_CURRENT_SOURCE_DIR}/libuuu ${LIBUSB_LIBRARY_DIRS} ${LIBZIP_LIBRARY_DIRS} ${LIBZ_LIBRARY_DIRS})
+link_directories(${CMAKE_CURRENT_SOURCE_DIR}/libuuu ${LIBUSB_LIBRARY_DIRS} ${LIBZIP_LIBRARY_DIRS})
 
 set(CLIST_EXECUTABLE ${CMAKE_CURRENT_SOURCE_DIR}/gen_txt_include.sh)
 set(generated_files_dir "${CMAKE_BINARY_DIR}/uuu/gen")
-- 
2.7.4

